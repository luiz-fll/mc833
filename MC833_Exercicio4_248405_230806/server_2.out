remoto: 127.0.0.1:6002
[SERVIDOR] Escutando em 127.0.0.1:6002
127.0.0.1:41644
GET / HTTP/1.0
Host: teste

127.0.0.1:41660
GET / HTTP/1.0
Host: teste

127.0.0.1:41672
GET / HTTP/1.0
Host: teste

127.0.0.1:41686
GET / HTTP/1.0
Host: teste

127.0.0.1:41700
GET / HTTP/1.0
Host: teste

127.0.0.1:41714
GET / HTTP/1.0
Host: teste

127.0.0.1:41730
GET / HTTP/1.0
Host: teste

127.0.0.1:41738
GET / HTTP/1.0
Host: teste

