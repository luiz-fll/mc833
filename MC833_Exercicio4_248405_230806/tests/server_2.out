remoto: 127.0.0.1:60002
[SERVIDOR] Escutando em 127.0.0.1:60002
127.0.0.1:52654
GET / HTTP/1.0
Host: teste

127.0.0.1:52656
GET / HTTP/1.0
Host: teste

127.0.0.1:52662
GET / HTTP/1.0
Host: teste

127.0.0.1:52668
GET / HTTP/1.0
Host: teste

127.0.0.1:52682
GET / HTTP/1.0
Host: teste

127.0.0.1:52698
GET / HTTP/1.0
Host: teste

127.0.0.1:52714
GET / HTTP/1.0
Host: teste

127.0.0.1:52718
GET / HTTP/1.0
Host: teste

127.0.0.1:52720
GET / HTTP/1.0
Host: teste

